src/wasm-binary.h

```diff
@@ -1553,7 +1553,6 @@ class WasmBinaryReader {
 
 public:
   WasmBinaryReader(Module& wasm,
-                   FeatureSet features,
```

Comment:
I think this means that, for the first time, we read things from `Module& wasm`, rather than consider it something empty for us to fill. So now it becomes sort of an in/out param, rather than just out.

How about still passing this in, and applying it immediately?

Reply:
Sure, I can do that.